tests/Unit/NumericalAlgorithms/Spectral/Test_Spectral.cpp

```diff
@@ -131,7 +131,6 @@ void test_weak_differentiation() {
               approx(weak_diff_matrix(i, l)));
       }
     }
-
```

Comment:
the commit message seems reasonable, but it doesn't seem to reasonably describe the commit :)

Reply:
Hah. oops. I guess that change got merged separately, aside from the newline delete. Dropped the commit.